zulipterminal/ui_tools/buttons.py

```diff
@@ -87,23 +87,35 @@ def keypress(self, size: Tuple[int, int], key: str) -> str:
 
 class HomeButton(TopButton):
     def __init__(self, controller: Any, width: int, count: int=0) -> None:
-        super().__init__(controller, 'All messages',
+        button_text = ("All messages   [" +
+                       keys_for_command("GO_BACK").pop() +  # FIXME
```

Comment:
I'm not sure I understand why there's a .pop() here?

Reply:
Here I'm using it like a 'pick one element'. The function returns a new set, and this is a simple way of picking one, since sets can't be indexed as they're not sequences. We could switch commands to be lists - order might be useful to prioritize keys - or even only have one key per command, but for now we don't need that: this works.

I didn't want to use a `", ".join` like with the search boxes, since they have a lot more space.

Of course, none of these commands have multiple keys assigned by default right now anyway!

Reply:
Sure. This is clarified. As you say perhaps the followup would be to make it list or Ordered Dict. (I remember Shreya has a PR which does that). LGTM :+1:

Reply:
Possibly; my emphasis was really that we can change this if we need to - so far, having the possibility of multiple keys is useful, and ordering doesn't seem a necessity.